index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
 
```

Comment:
<h3 hidden>Click <a href="https://www.google.com">here</a> to leave the site.</h3>

Reply:
This is a good addition.

Reply:
Well now I'm talking to myself. Where is the "Commit Suggestion" button???